src/commands/github-bot-delete-bot-comments.yml

```diff
@@ -10,6 +10,10 @@ parameters:
   really-delete-comments:
     type: boolean
     default: true
+  skip-last:
+    type: boolean
+    description: Update the last comment from this bot with this title instead of creating a new one
```

Comment:
I think you copypastaed a bit :). I think the description should be something like "if true, delete every comment except the last one"

Reply:
haha definitely did! Will update 👍